packages/ember-glimmer/lib/ember-views/target-action-support.js

```diff
@@ -0,0 +1,217 @@
+import { Mixin } from 'ember-metal/mixin';
```

Comment:
the only key difference is in the way the action is resolved either from `attrs.` or the `ARGS` symbol on the component.

That said, I get what you're saying, but I'm not sure how do make the change without modifying the API of the old `view_target_action_support.js` mixin. If I move the `send()` and `sendAction()` into `target-action-support.js` then `view_target_action_support` will also have those methods, even though in glimmer1 they're only added at the `component.js` level, where `view_target_action_support is mixed-in.

This means that anyone manually mixing-in `view_target_action_support.js` will get more than they bargained for. If this is fine though, I'll go ahead and do that